app/assets/sass/uswds/_uswds-theme-custom-styles.scss

```diff
@@ -378,28 +378,29 @@ td.table-empty-message {
 }
 
 .job-table {
-   .usa-table {
-      thead th.file-name {
-         width: 25%;
-      }
-      thead th.template {
-         width: 20%;
-      }
-      thead th.time-sent {
-         width: 30%;
-      }
-      thead th.sender {
-         width: 15%;
-      }
-      thead th.\#-of-recipients {
-         width: 5%;
-      }
-      thead th.report {
-         width: 5%;
-      }
-      th {
-         padding: 0.5rem 0.5rem;
-      }
+   width: 100%;
```

Comment:
These css changes is to help fix the column spacing. If a user uploads a csv with a long file name, the spacing will look decent, instead of squished like it did before

(image of squished table)
![image](https://github.com/GSA/notifications-admin/assets/53159604/dfefe613-5b5b-4024-89df-da376b43f6b1)

Reply:
Nice, thank you!